core/base/inc/TApplication.h

```diff
@@ -64,6 +64,7 @@ class TApplication : public TObject, public TQObject {
    Bool_t             fNoLogo;          //Do not show splash screen and welcome message
    Bool_t             fQuit;            //Exit after having processed input files
    Bool_t             fUseMemstat;      //Run with TMemStat enabled
+   Bool_t             fUsePedantic;     //Run with --pedantic enabled
```

Comment:
Maybe `fPedanticInterp`? Because in `TApplication` context, "pedantic" doesn't mean much.